interrupts/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+//! Interrupts
```

Comment:
Putting the name here isn't important; The docs will already say this is for the interrupts crate